src/feedback/data/sagas.js

```diff
@@ -48,6 +48,11 @@ export function* handleMessages(messages, clearExistingMessages) {
     yield put(clearMessages());
   }
 
+  if (window.location.search.indexOf('message') > 0) {
+    const message = new URLSearchParams(window.location.search).get('message');
+    yield put(addMessage('message', `${message}`, {}, MESSAGE_TYPES.ERROR));
```

Comment:
Where are you using this code 'message'? Maybe you can use a more descriptive word than just 'message', what type of message? You should also update the sagas.test file.

Reply:
i'm using it here
well it is meant to be generic on purpose, i guess i could say error_message if that's clearer since i did make the message_type error